README.md

```diff
@@ -16,6 +16,8 @@ This repository is forked on https://github.com/jjx3455/solar_berlin and https:/
 - (Alternatively, skip downloading the tiles and use the pre-processed GeoTIFFs available as "converted.zip" https://drive.google.com/drive/folders/1zJGu6x-S13IBi_N0VGynjAypKGTq6JSC)
 - (Alternatively 2, skip the extraction too and use the sample data available as "selected_512.zip" from https://drive.google.com/drive/folders/1zJGu6x-S13IBi_N0VGynjAypKGTq6JSC)
 
+- In order to run the Mask R CNN model, additionally install Detectron2: https://detectron2.readthedocs.io/en/latest/tutorials/install.html
```

Comment:
I am not sure if we want this part here, or in the requirements. I would naturally have it in the requirements. I am also open to have a sub-requirement for the mask R folder.

Reply:
I have moved the models to subfolders as discussed. I have not moved the unet run-files, but you could do that if you like, or leave them where they are. I have kept the main Readme - I suggest we move the model specific parts to the subfolders once we agree what the main Readme should contain.